plugins/org.eclipse.n4js/src/org/eclipse/n4js/external/NpmCLI.java

```diff
@@ -172,7 +172,7 @@ private LibraryChange installUninstall(MultiStatus batchStatus, File installPath
 			actualChangeType = LibraryChangeType.Removed;
 		}
 
-		if (packageProcessingStatus != null && packageProcessingStatus.isOK() && batchStatus.isOK()) {
+		if (packageProcessingStatus != null && packageProcessingStatus.isOK()) {
```

Comment:
Maybe keep the check here, but also add the check in the calling loop,
we want to fail fast. Otherwise user waits an hour to end up in a broken state anyway.